### DISCUSSION TITLE
Set dpi for images

### DISCUSSION THREAD

**Discussion by sherafatinia** [spatie/laravel-medialibrary#3304] [Q&A]
### Discussed in https://github.com/spatie/laravel-medialibrary/discussions/3209

<div type='discussions-op-text'>

<sup>Originally posted by **sherafatinia** March 15, 2023</sup>
Hi
Uploaded photos are saved with 96 dpi, I didn't find a way to save photos with 72 dpi, how can I do it?</div>

**Comment by hurradieweltgehtunter** [comment]
bump.
I'd love to have this feature.

So far, I'm trying to reduce the file size by resizing an uploaded image. While this does the job, it still can be optimized by setting the dpi. In my case, my app requires only 72dpi images. Uploading a 300dpi image is still possible, but it's totally unnecessary to store it in 300dpi.
